source/documentation/deploying_services/postgresql.md

```diff
@@ -233,6 +233,35 @@ To upgrade, you must set up a new service and migrate your app data.
 
 You cannot currently downgrade your service plan.
 
+### Reboot a PostgreSQL service instance
+
+You can [reboot](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_RebootInstance.html) [external link] your PostgreSQL service instance to:
+
+- try to fix a problem with your service instance
+- test how your app behaves during a service instance failure
+
+To reboot your service instance, you must have at least [space developer](https://docs.cloud.service.gov.uk/orgs_spaces_users.html#space-developer) permissions or higher in the [space](orgs_spaces_users.html#spaces) that hosts your service instance.
```

Comment:
We should fix the links:
- `/orgs_spaces_users.html#space-developer`
- `/orgs_spaces_users.html#spaces`